### DISCUSSION TITLE
Mulitple download function

### DISCUSSION THREAD

**Discussion by nothingness6** [LykosAI/StabilityMatrix#524] [Ideas]
Downloading one by one is the pain in the ass. Please add multiple download function.

**Comment by mohnjiles** [comment]
Could you be more specific? What would you like to install multiple of at the same time? Models? Packages? Updates?

**Reply by nothingness6** [reply]
My apologies. I mean the model browser.
![SM_model browser](https://github.com/LykosAI/StabilityMatrix/assets/105210258/0403de61-f55d-4c18-a424-7fad31506608)

**Comment by Targren** [comment]
Especially LoRAs, some creators use different "versions" to create "sets" (for example, [Edg's Royal Gowns](https://civitai.com/models/61135/royal-gowns-an-edg-collection).  That's a use case that shows up quite a lot for me.
